File: app/src/main/java/com/pavelprymak/propodcast/presentation/adapters/SearchPodcastClickListener.java
package com.pavelprymak.propodcast.presentation.adapters;

import android.view.View;

import com.pavelprymak.propodcast.model.network.pojo.search.ResultsItem;

public interface SearchPodcastClickListener {
    void onPodcastItemClick(String podcastId);

    void onPodcastMoreOptionsClick(ResultsItem podcastItem, View v);
}

File: app/src/main/java/com/pavelprymak/propodcast/presentation/adapters/SearchPodcastClickListener.kt
package com.pavelprymak.propodcast.presentation.adapters

import android.view.View

import com.pavelprymak.propodcast.model.network.pojo.search.ResultsItem

interface SearchPodcastClickListener {
    fun onPodcastItemClick(podcastId: String)

    fun onPodcastMoreOptionsClick(podcastItem: ResultsItem, v: View)
}
